content/04.network_applications.md

```diff
@@ -157,25 +157,25 @@ Applications that use deep learning techniques have used node2vec [@doi:10.1093/
 Zong et al. used a node2vec-like model to predict drug-target associations [@doi:10.1093/bioinformatics/btx160].
 The authors constructed a disease-target-disease network using drug centered databases: Drugbank [@doi:10.1093/nar/gkm958] and Diseasome [@doi:10.1073/pnas.0701361104].
 Next, the authors applied a random walk to the graph and trained a skip-gram model to generate a low dimensional representation of the graph.
-Lastly, the authors construct a similarity metric that used this space to rank how similar drugs are to their targets [@doi:10.1093/bioinformatics/btx160].
+Lastly, the authors constructed a similarity metric that used this space to rank how similar drugs are to their targets [@doi:10.1093/bioinformatics/btx160].
 A limitation to this approach is that their graph is missing information such as pharmacological class or drug chemical structure that could improve prediction performance.
 Overall, deep learning provides a robust set of techniques that have been shown to outperform most linear approaches in this context [@doi:10.1186/s12859-019-3284-5; @doi:10.1145/3307339.3342161].
 
 Applications that discover new properties of drugs have benefited from using knowledge graphs as a resource.
 Most methods to date use matrix factorization and deep learning techniques to produce a low-dimensional representation.
-Due to the success of deep learning [@doi:10.1186/s12859-019-3284-5; @doi:10.1145/3307339.3342161] much of the field's focus has shifted to these techniques; however, a possible extension is to use an ensemble of deep learning techniques and linear methods to improve performance.
-Plus, another area of exploration is to incorporate information such as pharmaceutical classes for drugs or chemical structure to improve detection power.
+Due to the success of deep learning [@doi:10.1186/s12859-019-3284-5; @doi:10.1145/3307339.3342161] much of the field's focus has shifted to these techniques; however, a possible improvement is to use an ensemble of deep learning techniques and linear methods to improve performance.
+Plus, another area of suggested exploration is to incorporate information such as pharmaceutical classes for drugs or chemical structure to improve detection power.
```

Comment:
```suggestion
Another potential avenue for future work would be to incorporate entity-specific hierarchical information or similarity information to improve detection power.
For drugs, this could include pharmaceutical classes or chemical structure similarities.
```
I think this is consistent with what you're saying here